FrontEnd/src/components/NavBar/NavBar.tsx

```diff
@@ -7,19 +7,25 @@ import { landingIcons } from '@icons';
 import styles from './NavBar.module.scss';
 import { Row } from 'react-bootstrap';
 import { useUser } from '@hooks';
+import { useRouter } from 'next/router';
 
 const NavBar: FunctionComponent = () => {
   const dispatch = useDispatch();
   const { data: user, logout } = useUser();
+  const router = useRouter();
   const itemProps = [
     {
       href: '/profile',
       label: 'Profile',
+      selected:
+        router.pathname.toLowerCase().slice(1) === 'Profile'.toLowerCase(),
```

Comment:
Why 'Profile'.toLowerCase() and not just 'profile'? Also, I don't think the first toLowerCase() is necessary since the path is '/profile'. Would probably be easier to read if you did:
router.pathname.slice(1) === 'profile'

Also, it seems that you reuse the same code in this line and line 28. Consider separating the first portion of it into a separate variable to use:
const descriptiveVarNameHere = router.pathname.toLowerCase().slice(1)

selected: descriptiveVarNameHere === 'Profile'.toLowerCase(),
selected: descriptiveVarNameHere === 'Logout'.toLowerCase(),

Reply:
you are absolutely right. Dunno why I did that lmao